<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>521</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>638</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1" colspan="4">
     <widget class="GraphicsView" name="graphicsView">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QCommandLinkButton" name="commandLinkButton_disableX11">
      <property name="text">
       <string>Auto disable X11 events</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QCommandLinkButton" name="commandLinkButton_fullscreen">
      <property name="text">
       <string>Fullscreen</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QCommandLinkButton" name="commandLinkButton_fit_to_screen">
      <property name="text">
       <string>Fit to screen</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>917</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd_unlisted_device">
    <property name="title">
     <string>Tools</string>
    </property>
    <widget class="QMenu" name="menuOrphans_Usb_Hid">
     <property name="title">
      <string>Orphans Usb Hid</string>
     </property>
    </widget>
    <addaction name="menuOrphans_Usb_Hid"/>
    <addaction name="actionFrom_VID_PID"/>
    <addaction name="actionExpert_Mode"/>
   </widget>
   <addaction name="menuAdd_unlisted_device"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidgetInputDevices">
   <property name="baseSize">
    <size>
     <width>350</width>
     <height>0</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Devices</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTabWidget" name="tabWidgetInputDevices">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>350</width>
         <height>0</height>
        </size>
       </property>
       <property name="tabPosition">
        <enum>QTabWidget::West</enum>
       </property>
       <property name="movable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionFrom_VID_PID">
   <property name="text">
    <string>From VID:PID</string>
   </property>
  </action>
  <action name="actionExpert_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Expert Mode</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>graphicsview.h</header>
   <slots>
    <slot>setFullscreen(bool)</slot>
    <slot>setFitToScreen(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>commandLinkButton_disableX11</tabstop>
  <tabstop>tabWidgetInputDevices</tabstop>
  <tabstop>graphicsView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>commandLinkButton_disableX11</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>disableX11(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>458</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_fullscreen</sender>
   <signal>clicked(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>setFullscreen(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>621</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_fit_to_screen</sender>
   <signal>clicked(bool)</signal>
   <receiver>graphicsView</receiver>
   <slot>setFitToScreen(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>774</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>770</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>commandLinkButton_fullscreen</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>fullscreen(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>584</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>589</x>
     <y>30</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>vidpidClicked()</slot>
  <slot>disableX11(bool)</slot>
  <slot>fullscreen(bool)</slot>
  <slot>setupDisplayView()</slot>
 </slots>
</ui>
